src/presentational-components/shared/table-toolbar-view-for-roles.js

```diff
@@ -0,0 +1,224 @@
+import React, { Fragment, useState } from 'react';
+import propTypes from 'prop-types';
+import { Table, TableHeader, TableBody, TableVariant } from '@patternfly/react-table';
+import TableToolbar from '@redhat-cloud-services/frontend-components/TableToolbar';
+import { Button, Pagination, EmptyStatePrimary } from '@patternfly/react-core';
+import { ListLoader } from './loader-placeholders';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import { selectedRows } from '../../helpers/shared/helpers';
+import Toolbar, { paginationBuilder } from './toolbar';
+import EmptyWithAction from './empty-state';
+import './table-toolbar-view.scss';
+
+export const TableToolbarViewForRoles = ({
+  isCompact,
+  borders,
+  columns,
+  rows,
+  toolbarButtons,
+  data,
+  actionResolver,
+  areActionsDisabled,
+  routes,
+  titlePlural,
+  titleSingular,
+  pagination,
+  filterValue,
+  isLoading,
+  emptyFilters,
+  setFilterValue,
+  checkedRows,
+  isSelectable,
+  fetchData,
+  setCheckedItems,
+  isCollapsible,
+  emptyProps,
+  filterPlaceholder,
+  rowWrapper,
+  filters,
+  isFilterable,
+  onShowMore,
+  showMoreTitle,
+  onFilter,
+  onChange,
+  value,
+  sortBy,
+  isExpandable,
+  onExpand,
+  hideFilterChips,
+  hideHeader,
+  noData,
+  noDataDescription,
+  ouiaId,
+  tableId,
+  containerRef,
+  onSort,
+}) => {
+  // eslint-disable-next-line no-unused-vars
+  const [opened, openRow] = useState({});
+  const onCollapse = (_event, _index, isOpen, { uuid }) =>
```

Comment:
@victoria-dos maybe I am missing something, but why do we need this? Is there any case when the table is collapsible? If not, we probably don't need to copy it from `table-toolbar-view.js`